ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -273,6 +264,72 @@ impl Blockstore {
         Ok(columns_purged)
     }
 
+    fn purge_range(&self, write_batch: &mut WriteBatch, from_slot: Slot, to_slot: Slot) -> bool {
```

Comment:
does it make sense to include `purge_special_columns_exact()` in here?

```
fn purge_range() {}
fn purge_range_special_columns(){}
fn do_purge_range(&self, write_batch: &mut WriteBatch, from_slot: Slot, to_slot: Slot, should_purge_special_columns: bool)
```

Reply:
good idea, i just passed through `PurgeType` instead of separate functions.